toolkits/global/packages/global-corporate-footer/demo/main.scss

```diff
@@ -3,3 +3,7 @@
 
 @import '../scss/10-settings/default';
 @import '../scss/50-components/corporate-footer';
+
+@import '../node_modules/@springernature/brand-context/default/scss/30-mixins/lists';
+
+@import '../node_modules/@springernature/brand-context/default/scss/60-utilities/container';
```

Comment:
You need to include the `brand-context` here, which will get dynamically included when you render the demo. This also means you don't need to define the mixin you are using as this will come with the `brand-context` include.

Follow the format found here: https://github.com/springernature/frontend-toolkits/blob/master/toolkits/nature/packages/nature-hero/demo/main.scss